uint8 port # Port number of the device 1-3
bool current_port_changed # True if the current port is connected
uint32 baudrate # Baudrate of the device 9600, 19200, 38400, 57600, 115200, 230400, 460800, 921600
---
bool success
std_msgs/String response
std_msgs/String error_message